src/main.js

```diff
@@ -13,6 +13,19 @@
     }
   }
 
+  function NoApiKeySet() {
+    return /*#__PURE__*/React.createElement('div', {
+        className: 'alert alert-warning'
+      },
+      /*#__PURE__*/React.createElement('span', {
+        className: 'fa fa-exclamation-triangle',
+        style: {
+          marginRight: '5px'
+        }
+      }),
+      'The Google API Key has not been set.');
+  }
```

Comment:
Why is this duplicate here and in `...control/YouTube/main.js`? Also looks like the output of a transpilation. Refactor.